Product/Production/Gateway/AdminGUIWebservices/src/main/java/gov/hhs/fha/nhinc/configadmin/ConfigAdmin.java

```diff
@@ -630,13 +643,21 @@ public SimpleCertificateResponseMessageType importToKeystore(ImportToKeystoreReq
         if (StringUtils.isBlank(alias) || null == request.getServerCert()) {
             return buildSimpleResponse(false, "Alias and Server Certificate are required.");
         }
+
+        PrivateKey privateKey = readPrivateKey(alias, getPasswordKeystore());
+        if (null == privateKey) {
+            return buildSimpleResponse(false, "Error no privatekey recorded for alias: " + alias);
+        }
+
+        tempKeystore = null;
+        if (!copyFile(FILE_JKS_GATEWAY, FILE_JKS_GATEWAY_NEW)) {
+            return buildSimpleResponse(false, "unable to make a copy of gateway-jks.");
+        }
+
         try {
             Certificate publicKey = CertificateUtil.createCertificate(request.getServerCert());
-            PrivateKey privateKey = CertificateUtil.getPrivateKeyEntry(getTempKeystore(), alias,
-                System.getProperty(CertificateManagerImpl.KEY_STORE_PASSWORD_KEY)).getPrivateKey();
-
-            if(null == publicKey || null == privateKey){
-                return buildSimpleResponse(false, "Error certificate does not have keypair: "+alias);
+            if (null == publicKey) {
+                return buildSimpleResponse(false, "Error server certificate is not upload.");
```

Comment:
"is not upload" -> "was not uploaded"
Remove the word "Error" to keep consistent with other messages.